src/System Application/App/File System/app.json

```diff
@@ -0,0 +1,75 @@
+{
+    "id":  "c9c54414-80c3-4cc9-98c6-589158882774",
+    "name":  "File System",
+    "publisher":  "Microsoft",
+    "brief":  "Enables access to external file systems from Business Central.",
+    "description":  "Enables access to external file systems from Business Central.",
+    "version":  "25.0.0.0",
+    "privacyStatement":  "https://go.microsoft.com/fwlink/?linkid=724009",
+    "EULA":  "https://go.microsoft.com/fwlink/?linkid=2009120",
+    "help":  "https://go.microsoft.com/fwlink/?linkid=2103698",
+    "url":  "https://go.microsoft.com/fwlink/?linkid=724011",
+    "logo":  "",
+    "dependencies":  [
+                        {
+                            "id":  "7e3b999e-1182-45d2-8b82-d5127ddba9b2",
+                            "name":  "DotNet Aliases",
+                            "publisher":  "Microsoft",
+                            "version":  "25.0.0.0"
+                        },
+                        {
+                            "id":  "872fe7e8-9893-40ae-ab94-c123ed30fdbd",
+                            "name":  "Extension Management",
+                            "publisher":  "Microsoft",
+                            "version":  "25.0.0.0"
+                        },
+                        {
+                            "id":  "de35f591-7216-4e60-8be1-1911d71a7fc2",
+                            "name":  "Telemetry",
+                            "publisher":  "Microsoft",
+                            "version":  "25.0.0.0"
+                        },
+                        {
+                            "id":  "0846d207-5dec-4c1b-afd8-6a25e1e14b9d",
+                            "name":  "Base64 Convert",
+                            "publisher":  "Microsoft",
+                            "version":  "25.0.0.0"
+                        },
+                        {
+                            "id":  "a4584a53-9345-458a-af20-d1df2fab7bd8",
+                            "name":  "Confirm Management",
+                            "publisher":  "Microsoft",
+                            "version":  "25.0.0.0"
+                        },
+                        {
+                            "id":  "e31ad830-3d46-472e-afeb-1d3d35247943",
+                            "name":  "BLOB Storage",
+                            "publisher":  "Microsoft",
+                            "version":  "25.0.0.0"
+                        }
+                    ],
+    "internalsVisibleTo":  [
+                               {
+                                   "id":  "fff6eee5-083f-4f07-9a49-e34f7e2aad77",
```

Comment:
you should not add the test app here.

Reply:
The base structure is copied from the Email Module. It is exactly the same there, how should I fix it?

Reply:
The correct pattern is to grant internals access to a test library app, which you have done here already so the fix is to remove the test app here and then fix the tests by refactoring to use public methods or adding what is missing in the test library and using that.

Reply:
Thanks for the explanation. I have restructured it.